packages/editor-ui/src/components/WorkflowSettings.vue

```diff
@@ -488,6 +490,7 @@ export default mixins(
 			this.closeDialog();
 
 			this.$externalHooks().run('workflowSettings.saveSettings', { oldSettings });
+			this.$telemetry.track('User updated workflow settings', { workflow_id: this.$store.getters.workflowId });
```

Comment:
It would be amazing to know how many users set `Save progress` to yes. Maybe not the others, but this one specifically I think useful.